src/styles/title.scss

```diff
@@ -0,0 +1,23 @@
+.main {
+  display: flex;
+  flex-direction: column;
+
+  &__title {
+    box-sizing: border-box;
+    position: flex;
+    display: inline;
+    font-family: "Edu NSW ACT Foundation", sans-serif;
+    font-weight: 500;
+    font-size: 60px;
+    margin: 15px auto 0;
+    box-shadow: -8px 10px teal, 10px -10px rgb(52, 42, 104);
+    border: 1px solid rgb(192, 192, 199);
+
+    transition: all 0.4s ease-in-out;
```

Comment:
Always add transition property (what styles will change smoothly). `All` is a default value, and it's hard for the browser to control all properties